public/src/Helpers.ts

```diff
@@ -277,11 +271,17 @@ export const userLink = (user: User): string => {
     return `https://github.com/${user.Login}`
 }
 
-export const userRepoLink = (course: Course, user: User): string => {
+export const userRepoLink = (user: User, course?: Course): string => {
```

Comment:
These helpers (userLink, UserRepoLink, and groupRepoLink) should be removed or changed to use information obtained from the updated GetRepositories() RPC method. (Maybe we can fix it later, once #942 has been merged.)

Reply:
Currently `GetRepositories` returns the repositories for a single user. We only use the returned repositories in the `CourseUtilityLinks` component.

For enrollments fetched for an entire course we currently need these helper methods to build the links. We *could* add `Repositories` to both the `Group` and `Enrollment` types, and preload the repositories. If we do this we could avoid using these helpers.

Reply:
Sounds reasonable. We should do it in a separate PR. Please create an issue to do it later.